hexagonal/src/main/java/com/iluwatar/hexagonal/domain/LotteryNumbers.java

```diff
@@ -85,11 +84,10 @@ public static LotteryNumbers create(Set<Integer> givenNumbers) {
    * @return numbers as comma separated string
    */
   public String getNumbersAsString() {
-    List<Integer> list = new ArrayList<>();
-    list.addAll(numbers);
     StringBuilder builder = new StringBuilder();
+    Iterator<Integer> iterator = numbers.iterator();
```

Comment:
Can't we directly use foreach loop and not depend on `NUM_NUMBERS`?

Reply:
I think so too. However, I'll accept this PR as an improvement and a step into the right direction.